src/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+
+const Button = ({ className, title }) => (
+  <button type="button" className={className}>{ title }</button>
+);
+
+Button.defaultProps = {
+  title: '',
+};
+
+Button.propTypes = {
+  className: PropTypes.string.isRequired,
+  title: PropTypes.string,
```

Comment:
- [ ]  Your button props still have some issue that's need to be fixed, please use valid protypes for title to fix the issue form the browser as it is shown bellow  
![image](https://user-images.githubusercontent.com/43877689/227341170-06f0b54e-433e-4a52-94ba-3d2c0855be5f.png)